docs/zh/latest/README.md

```diff
@@ -271,27 +271,27 @@ CentOS 7, Ubuntu 16.04, Ubuntu 18.04, Debian 9, Debian 10, macOS, **ARM64** Ubun
 
 #### Apache APISIX 的优势
 
-| **功能**                              | **Apache APISIX**                       | **KONG**               |
+| **功能**                               | **Apache APISIX**                       | **KONG**               |
 | :------------------------------------ | :-------------------------------------- | :--------------------- |
-| 项目归属                              | Apache 软件基金会                       | Kong Inc.              |
-| 技术架构                              | Nginx + etcd                            | Nginx + postgres       |
-| 交流渠道                              | 微信群、QQ 群、邮件列表、GitHub、meetup | GitHub、论坛、freenode |
-| 单核 QPS (开启限流和 prometheus 插件) | 18000                                   | 1700                   |
-| 平均延迟                              | 0.2 毫秒                                | 2 毫秒                 |
-| 支持 Dubbo 代理                       | 是                                      | 否                     |
-| 配置回滚                              | 是                                      | 否                     |
-| 支持生命周期的路由                    | 是                                      | 否                     |
-| 插件热更新                            | 是                                      | 否                     |
-| 用户自定义：负载均衡算法、路由        | 是                                      | 否                     |
+| 项目归属                               | Apache 软件基金会                         | Kong Inc.              |
+| 技术架构                               | Nginx + etcd                            | Nginx + postgres       |
+| 交流渠道                               | 微信群、QQ 群、邮件列表、[GitHub](https://github.com/apache/apisix/issues)、[Slack](https://join.slack.com/t/the-asf/shared_invite/zt-mrougyeu-2aG7BnFaV0VnAT9_JIUVaA)、meetup | GitHub、论坛、freenode |
```

Comment:
Hi, I just generated a new link: https://join.slack.com/t/the-asf/shared_invite/zt-nggtva4i-hDCsW1S35MuZ2g_2DgVDGg

Slack invitation link may get expired :)